### PR TITLE
Create export csv option

### PR DESCRIPTION
**Pull request by MexsonFernandes**
For now, I have added export JSON to CSV. I have used module https://www.npmjs.com/package/export-from-json to convert the JSON to relevant extension (for now CSV).

### CODE REVIEW

File: src/components/TodoList.jsx [hunk 1]
```diff
@@ -54,6 +55,14 @@ export const TodoList = () => {
     setTasks(tasks.concat([{ value: '', completed: false, id: uuid() }]));
   };
 
+  const exportData = () => {
+    const data = tasks;
+    const fileName = 'todo';
+    const exportType = 'csv';
+    console.log(tasks);
```
Comment (markmhx) [inline src/components/TodoList.jsx:62]: Can be removed

**Comment by markmhx** [comment]
Very cool! Works for me

Screen Shot 2020-10-06 at 15 53 48

File: src/components/TodoList.jsx [hunk 2]
```diff
@@ -65,6 +74,13 @@ export const TodoList = () => {
     />
   ));
 
+  const getDownload = () => {
+    if (loading) {
+      return '';
+    }
+    return 'Export as csv';
```
Comment (markmhx) [inline src/components/TodoList.jsx:81]: I'd suggest uppercase for "CSV" here

File: src/components/TodoList.jsx (hunk 1 above)
Comment (MexsonFernandes) [reply src/components/TodoList.jsx:62]: sure I am fixing this

File: src/components/TodoList.jsx (hunk 2 above)
Comment (MexsonFernandes) [reply src/components/TodoList.jsx]: Done

**Review by markmhx — APPROVED** [review]
LGTM!

**Review by kyranjamie — APPROVED** [review]

**Comment by markmhx** [comment]
Hmm now it says "This branch cannot be rebased due to conflicts". @kyranjamie thoughts?

**Comment by kyranjamie** [comment]
It means there are conflicts that need to be resolved. 

Git has instructions, but along the line of:

```sh 
git fetch 
git rebase -i origin/master
```

**Comment by MexsonFernandes** [comment]
I tried, not able to rebase it.

**Comment by MexsonFernandes** [comment]
Created a new pull request #54

**Pull request closed without merging**